crates/version/src/tests.rs

```diff
@@ -0,0 +1,60 @@
+use crate::version_str;
+
+#[test]
+fn version_info_contains_crate_version() {
+    let info = version_str();
+    let expected = option_env!("CARGO_PKG_VERSION").unwrap_or("<missing>");
```

Comment:
I think this should be unconditional, `env!`, if it's not specified, the build script must not have succeeded, in which case, this doesn't run